arch/x86_64/mcount.S

```diff
@@ -1,8 +1,54 @@
 /* argument passing: %rdi, %rsi, %rdx, %rcx, %r8, %r9 */
 /* return value: %rax */
 /* callee saved: %rbx, %rbp, %rsp, %r12-r15 */
-/* stack frame (with -pg): return addr = 8(%rbp), prev fp = 0(%rbp) */
-/* stack frame (with -fentry): return addr = (%rsp), prev fp = %rbp */
+/* stack frame (with -pg): parent addr = 8(%rbp), child addr = 56(%rsp) */
```

Comment:
@namhyung , What about adding the explanation about #92 at [fentry.S:27~](https://github.com/namhyung/uftrace/blob/master/arch/x86_64/fentry.S#L27-L30) and [mcount.S:27~](https://github.com/namhyung/uftrace/blob/master/arch/x86_64/mcount.S#L27-L32) ?